app/home/controllers.js

```diff
@@ -5,6 +5,10 @@ const permissions = require('../../common/middleware/permissions')
 function dashboard(req, res) {
   const userPermissions = get(req.session, 'user.permissions')
 
+  if (permissions.check('allocations:view', userPermissions)) {
```

Comment:
I'm just working on supporting this on the dashboard so you can probably remove this commit from this PR. Maybe keep a forked branch in case we want to merge them still afterwards.